CHANGES.md

```diff
@@ -1,6 +1,7 @@
 ## 0.x.y
 
 * Removed unused TLS options from the k8s storage plugin config
+* An `ip` option in admin config to set ip to listen on
```

Comment:
Prefer that this start with a verb. Something like:
> Add an `ip` option to `admin` configuration so that access to the admin server may be constrained